src/main/java/Bean.java

```diff
@@ -173,6 +169,11 @@ public static void main(String[] args) {
                 printErrorMessage();
             }
         }
+    }
+
+    public static void main(String[] args) {
+        greet();
+        processUserInput();
         // exit because: userInput.equals("bye") || Task.getNumberOfTasks >= MAX_LIST_COUNT
         exit();
```

Comment:
Would it be more appropriate to move or consolidate exit handling within processUserInput() to maintain a clear separation of duties?